pandas/io/formats/style.py

```diff
@@ -626,7 +635,13 @@ def _apply(self, func, axis=0, subset=None, **kwargs):
         self._update_ctx(result)
         return self
 
-    def apply(self, func, axis=0, subset=None, **kwargs):
+    def apply(
+        self,
+        func: Callable[[FrameOrSeries], FrameOrSeries],
+        axis: Axis = 0,
+        subset: Optional[_IndexSlice] = None,
+        **kwargs
+    ):
```

Comment:
OK, but may need to change to generic self https://mypy.readthedocs.io/en/latest/generics.html#generic-methods-and-generic-self because of MyStyler subclass when adding type annotations to functions in this module in future pass. not sure until look at this.

The only reason I added type annotations to the arguments of apply was to ensure they were compatible with _ApplyArgs otherwise this pass would have been to fix check-untyped-def errors only